sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/TriggerBindings/Resolver/SignalRRequestResolver.cs

```diff
@@ -19,8 +19,7 @@ internal class SignalRRequestResolver : IRequestResolver
     {
         private readonly bool _validateSignature;
 
-        // Now it's only used in test, but when the trigger started to support AAD,
-        // It can be configurable in public.
+        // Now it's only used in test
         internal SignalRRequestResolver(bool validateSignature = true)
```

Comment:
Should we set this param to false instead for the AAD path? Or do we not know if we are in AAD path without checking the access keys?

Reply:
We don't know if we are in AAD path without checking the access keys. The request resolver is actually a singleton initialized at the beginning, but the SignalR connections are resolved at runtime.